FastSurferCNN/generate_hdf5.py

```diff
@@ -224,7 +224,7 @@ def create_hdf5_dataset(self, plane='axial'):
                         help="Turn on to add cortex mask for hires-processing.")
     parser.add_argument('--processing', type=str, default="aparc", choices=["aparc", "aseg", "none"],
                         help="Use aseg, aparc or no specific mapping processing")
-    parser.add_argument('--sizes', nargs='+', type=int, default=256)
+    parser.add_argument('--sizes', nargs='+', type=int, default=256, help="Resolutions of images in the dataset. Default: 256")
```

Comment:
This is not resolution, but image size (which in FreeSurfer depends on the resolution). But here this is just the image size.